cornflow-server/cornflow/cli/init_cornflow_service.py

```diff
@@ -148,12 +148,22 @@ def init_cornflow_service():
             f"Starting cornflow + {os.getenv('EXTERNAL_APP_MODULE', '../external_app')}"
         )
         os.chdir("/usr/src/external_app")
+
+        if register_key():
+            github_host = os.getenv("github_host", None)
```

Comment:
`"github_host"` -> better in capital letters, right?

`github_host = os.getenv("GITHUB_HOST", None)`

Reply:
Pycharm prefers if they are small caps